### PR TITLE
Ref issue 12 add script remark.

### PR DESCRIPTION
**Pull request by tai271828**
This commit is according to the discussion of PR 11 to add the PR
description to be the script remark. This commit is related to issue 10
of project 1 as well. I also tweak the format and logic to show the
usage description to make this script be more readable and maintainable.

### CODE REVIEW

File: bin/admin/bootstrap-cluster.sh [hunk 1]
```diff
@@ -1,5 +1,82 @@
 #!/bin/bash
 
+usage_description() {
```
Comment (tai271828) [inline bin/admin/bootstrap-cluster.sh:3]: A function to collect the detailed description. Only text section is here.

File: bin/admin/bootstrap-cluster.sh [hunk 2]
```diff
@@ -72,8 +149,8 @@ build_nfs_server() {
   echo "building a NFS server...done!"
 }
 
-usage="Usage : $0 <node_type>"
-usage="$usage\n node_type: master, client, nfs_server"
+usage="Usage : $0 <node_type>\n"
```
Comment (tai271828) [inline bin/admin/bootstrap-cluster.sh:152]: This change is used for `printf` to has a better looking of the output format.

File: bin/admin/bootstrap-cluster.sh [hunk 3]
```diff
@@ -122,14 +201,9 @@ case $node_type in
   *)
   # unknown node_type
   echo "unknown node_type"
-  echo $usage
+  printf "$usage"
   exit
   ;;
 esac
 
-if [ -n "$help" ]; then
```
Comment (tai271828) [inline bin/admin/bootstrap-cluster.sh:130]: If there is a help option, show the detailed usage description and stop this script immediately, so I don't need to wait for the rest of this script is run.

**Comment by tai271828** [comment]
This commit is trivial and does not really touch any implementation of the other components. The risk of regression is low, so I will self-approve to merge this PR.

**Comment by yungyuc** [comment]
I dislike to print the usage to terminal.  It's too heavy weight.  My original comment on #11 was ["add it to the script remark"](https://github.com/solvcon/solvcon-gce/pull/11#issuecomment-246109400).  I didn't mean to let a command-line option to print the text to users.  I hope users to read the script itself rather than the output of the script.  

Could you please change it to use code remark?

Sorry for the outdated review.  I will be more responsive.  But please do wait for my review before merge.

**Comment by tai271828** [comment]
PR https://github.com/solvcon/solvcon-gce/pull/15 was created to make it be a part of and only a part of the script remark. Please review it. Thanks!

**Pull request merged**